cli/src/commands/split.rs

```diff
@@ -244,3 +241,39 @@ The remainder will be in the second commit.
     tx.finish(ui, format!("split commit {}", target_commit.id().hex()))?;
     Ok(())
 }
+
+impl SplitArgs {
```

Comment:
nit: might be better to move `impl` closer to the type definition?

Reply:
I was trying to keep cmd_split towards the top of the file, but maybe that doesn't matter. Done.